client/src/App.jsx

```diff
@@ -33,10 +38,54 @@ class App extends React.Component {
       })
   }
 
+  computeTotalRating() {
+    let averageRatings = [];
+
+    for(let review of this.state.reviews) {
+      let averageRating = 0;
+      for(let rating in review.ratings) {
+        rating = review.ratings[rating];
+        averageRating += rating;
+      }
+      averageRating = averageRating / 6.0;
```

Comment:
Instead of hard coding 6.0 ratings, might be cleaner to use `Object.keys(review.ratings).length` or `review.ratings.length`